cognite_toolkit/cdf_tk/load/_base_loaders.py

```diff
@@ -255,6 +255,14 @@ def deploy_resources(
         self.build_path = path
         filepaths = self.find_files(path)
 
+        def sort_key(p: Path) -> int:
+            if result := re.findall(r"^(\d+)", p.stem):
+                return int(result[0])
+            else:
+                return len(filepaths)
+
+        filepaths = sorted(filepaths, key=sort_key)

```

Comment:
Would be good with a comment on what you are trying to achieve here. It is not immediately obvious why you need a custom sort.